pints/tests/test_transformation.py

```diff
@@ -0,0 +1,734 @@
+#!/usr/bin/env python3
+#
+# Tests Transformation functions in Pints
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+from __future__ import division
+import unittest
+import pints
+import pints.toy
+import numpy as np
+
+
+class TestTransformation(pints.Transformation):
```

Comment:
An obvious extra test in all these methods could be `assertEqual(p, detransform(transform(p)))` ?
It _almost_ follows from what we have now, but I think having this too would catch any issues where the shape slightly changes or anything like that?

Reply:
Added.